internal/api/credential.go

```diff
@@ -8,10 +8,10 @@ import (
 )
 
 var credentialArtifactDefinitions = []*ArtifactDefinition{
-	{"massdriver/aws-iam-role"},
-	{"massdriver/azure-service-principal"},
-	{"massdriver/gcp-service-account"},
-	{"massdriver/kubernetes-cluster"},
+	{"massdriver/aws-iam-role", "AWS IAM Role", ""},
```

Comment:
Why are these changing? This is specifically for integrating credentials definitions not _all artifacts_ … we also have a mediocre inflector that does this. If we need it elsewhere we should bring it up from the arttdef table UI component as a strict function Titleize()

but also still curious why credential struct being modified wrt listing _all_ artifacts